src/beast/test/asio/streambuf.cpp

```diff
@@ -0,0 +1,296 @@
+//
```

Comment:
You have to run *`b2`* from the root of the beast tree to build the standalone tests.